src/index.js

```diff
@@ -38,9 +39,14 @@ module.exports = (
     sourceMap = false,
     sourceMapRegister = true,
     watch = false,
-    v8cache = false
+    v8cache = false,
+    quiet = false
```

Comment:
We should probably document this in the README.

Reply:
Thanks, done! ✅